src/main/kotlin/Main.kt

```diff
@@ -1,3 +1,40 @@
+import org.jfree.chart.ChartFactory
+import org.jfree.chart.ChartPanel
+import org.jfree.data.category.DefaultCategoryDataset
+import resolver.Resolver
+import javax.swing.JFrame
+
+
 fun main(args: Array<String>) {
-    print("Yeah rock!")
+    val resolver = Resolver()
+    val countPlayersWithoutAgency = resolver.getCountWithoutAgency()
+    val bestDefender = resolver.getBestScorerDefender()
+    val expensiveGermanPlayer=resolver.getTheExpensiveGermanPlayerPosition()
+    val rudestTeam = resolver.getTheRudestTeam()
+
+    println("1. Количество игроков, интересы которых не представляет агенство -- $countPlayersWithoutAgency")
+    println("2. Защитник ${bestDefender.first} набрал большее число голов ${bestDefender.second}")
+    println("3. Самый дорогой немецкий игрок играет на позиции $expensiveGermanPlayer")
+    println("4. Команда с наибольшим средним числом красных карточек на одного игрока ${rudestTeam.name}")
```

Comment:
как вариант чтобы по 5 раз не писать resolver - функция let

```suggestion
    Resolver(players).let {
        println(it.getCountWithoutAgency())
        println(it.getBestScorerDefender())
        println(it.getTheExpensiveGermanPlayerPosition())
        println(it.getTheRudestTeam())
    }
```

Reply:
done🫡